ros_gz_bridge/test/utils/gz_test_msg.cpp

```diff
@@ -92,6 +92,21 @@ void compareTestMsg(const std::shared_ptr<ignition::msgs::Float> & _msg)
   EXPECT_FLOAT_EQ(expected_msg.data(), _msg->data());
 }
 
+void createTestMsg(ignition::msgs::Float_V & _msg)
+{
+  ignition::msgs::Float msg;
+  createTestMsg(msg);
+  _msg.add_data(msg.data());
+}
+
+void compareTestMsg(const std::shared_ptr<ignition::msgs::Float_V> & _msg)
+{
+  ignition::msgs::Float_V expected_msg;
+  createTestMsg(expected_msg);
+
+  EXPECT_FLOAT_EQ(expected_msg.data()[0], _msg->data()[0]);
```

Comment:
Should we double-check that all values are the same and the sizes are similar? Same in the other `compareTestMsg`.

Reply:
yep, updated test [21f82d1](https://github.com/gazebosim/ros_gz/pull/306/commits/21f82d1a3b07950e4369fc610e48b1820a6a00b1)